litellm/main.py

```diff
@@ -4909,7 +4909,6 @@ def speech(
     litellm_call_id: Optional[str] = kwargs.get("litellm_call_id", None)
     proxy_server_request = kwargs.get("proxy_server_request", None)
     model_info = kwargs.get("model_info", None)
-    metadata = kwargs.get("metadata", {})
```

Comment:
`metadata` has already been defined in the function signature. Getting from `kwargs` effectively overwrites it.